toolchain/gcc.py

```diff
@@ -52,12 +52,12 @@ def setUp(self):
                              'gmp-devel', 'glibc-devel', 'mpfr-devel',
                              'makeinfo', 'texinfo', 'mpc-devel'])
         else:
-            packages.extend(['glibc-static', 'guile-devel', 'elfutils-devel',
+            packages.extend(['glibc-static', 'elfutils-devel',
```

Comment:
This looks like an incremental patch on top of previous changes. Can you include all the changes (previous and current) into a single commit?